scripts/add_link_slope_to_network.py

```diff
@@ -35,12 +34,18 @@
                             help='Output directory for the updated network',
                             required=True)
 
+    arg_parser.add_argument('-sj',
+                            '--save_jsons',
```

Comment:
I'd call this `save_validation_jsons` or just `save_validation`; network and schedule can be saved to json to so that will make the param less vague